### ISSUE TITLE
Loss cannot be decreased

### ISSUE THREAD

**Issue by su161021** [lucidrains/DALLE-pytorch#419]
Hello, I trained with the original parameters and found that the loss would not decrease if it was reduced to 4. The test effect was very poor, I tried to deepen the depth, but found that the effect was still not good. Is there any solution?
